### PR TITLE
[enterprise-4.16] OSDOCS-6182: add RN entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-6182

Link to docs preview: https://75640--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-coredns-improved-egress-firewall

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@melvinjoseph86 could you PTAL at this docs PR for QE ack? this is the OCP 4.16 release note entry for improved coredns integration with egress firewall

**Comment by melvinjoseph86** [comment]
Rest fine.
/label qe-approved

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -113,6 +113,13 @@ With this release, IP addresses and ranges are handled more efficiently in NSGs
 
 With this release, migration from the OpenShift SDN network plugin to OVN-Kubernetes is now supported on Nutanix platforms. For more information, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc#nw-ovn-kubernetes-migration-about_migrate-from-openshift-sdn[Migration to the OVN-Kubernetes network plugin].
 
+[id="ocp-4-16-coredns-improved-egress-firewall"]
+==== Improved integration between CoreDNS and egress firewall (Technology Preview)
+
+With this release, OVN-Kubernetes uses a new `DNSNameResolver` custom resource to keep track of DNS records in your egress firewall rules, and is available as a Technology Preview. This custom resource supports this use of both wildcard DNS names and regular DNS names and allows access to DNS names regardless if the IP addresses associated with it change.
```
Comment (kcarmichael08) [inline release_notes/ocp-4-16-release-notes.adoc:119]:
```suggestion
With this release, OVN-Kubernetes uses a new `DNSNameResolver` custom resource to keep track of DNS records in your egress firewall rules, and is available as a Technology Preview. This custom resource supports the use of both wildcard DNS names and regular DNS names and allows access to DNS names regardless of the IP addresses associated with its change.
```

**Review by kcarmichael08 — COMMENTED** [review]
Small comment/suggestion; otherwise lgtm!

**Pull request merged**